Mathlib/Tactic/Widget/GCongr.lean

```diff
@@ -3,6 +3,7 @@ Copyright (c) 2023 Patrick Massot. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Patrick Massot
 -/
+import Mathlib.Algebra.ModEq
```

Comment:
This import is needed so the name `Int.ModEq` is recognised.

Reply:
Am I right that the widget works fine if `Int.ModEq` is unknown? In that case I'm probably not in favour of adding the import which almost doubles the number of files this depends on.

(although, I also think it doesn't matter too much since it's a widget and will not be imported anywhere)

Reply:
Good question - I don't know!

Reply:
Pretty sure that's how it works, as `goalType.isAppOf` works just on `Expr`.

Reply:
:+1: I have reverted that change.